packages/commonjs/src/resolve-require-sources.js

```diff
@@ -108,7 +108,7 @@ export function getRequireResolver(extensions, detectCyclesAndConditional, curre
         setInitialParentType(parentId, parentMeta.initialCommonJSType);
         await Promise.all(
           parentRequires.map(({ resolved, isConditional }) =>
-            analyzeRequiredModule(parentId, resolved, isConditional, this.load)
+            analyzeRequiredModule(parentId, resolved, isConditional, this.load.bind(this))
```

Comment:
Fair enough, but Rollup already this-binds all context functions. This is by design, but admittedly, this is not well-communicated.

Reply:
ah, okay. this issue appeared while using it with vitejs.

Reply:
Interesting. So I assume this happened in dev mode? Then this would mean that Vite does *not* this.bind. Maybe an issue to raise with Vite. But this would also make it harder to write a test as you would need to test with a fake plugin context.

Reply:
Also, Andrew will not like that you skipped the PR template where you should have added all this information.

Reply:
I think so too ^^ 😅

Reply:
I added it now